internal/pkg/workspace/workspace.go

```diff
@@ -188,6 +188,16 @@ func (ws *Workspace) WorkloadExists(name string) (bool, error) {
 	return exists, nil
 }
 
+// EnvironmentsExist returns true if any environment is managed in the workspace.
+func (ws *Workspace) EnvironmentsExist() (bool, error) {
```

Comment:
Maybe `AnyEnvironmentExist` or `HasEnvironment` by reading the comment?

Reply:
done!